index.html

```diff
@@ -12003,6 +12003,45 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="property" id="aria-labelsynonyms">
+			<pdef>aria-labelsynonyms</pdef>
+			<div class="property-description">
+				<p><a>Defines</a> an optional <a data-cite="HTML/common-microsyntaxes.html#comma-separated-tokens">set of comma separated tokens</a> that provides alternate activation labels for the the current element. See related <pref>aria-label</pref> and <pref>aria-labelledby</pref>.</p>
+				<p>The purpose of <pref>aria-labelsynonyms</pref> is to allow a user of assistive technology such as voice activation software to activate controls using any one of several possible names. Other uses may include "type-ahead" function in a keyboard access utility or search function. For example, a button that closes a dialog may be labeled "close" for a screen reader, but sighted user of voice activation may say "tap x" or "tap dismiss."</p>
```

Comment:
minor wording updates.  specifically, to be consistent with the first instance of saying "possible names" vs then switching to say a 'label of "close"... for a screen reader"

```suggestion
				<p>The purpose of <pref>aria-labelsynonyms</pref> is to allow a user of assistive technology, such as voice activation software, to activate controls using any one of several possible names. Other uses may include "type-ahead" function in a keyboard access utility or search function. For example, a button that closes a dialog may have an accessible name of "close" for a screen reader, but a sighted user of voice activation may say "tap x" or "tap dismiss."</p>
```